packages/suite-desktop-core/src/modules/trezor-connect.ts

```diff
@@ -91,7 +91,10 @@ export const initBackground: ModuleInitBackground = ({ mainThreadEmitter, store
     const unregisterProxy = createIpcProxyHandler(ipcMain, 'TrezorConnect', ipcProxyOptions);
 
     const onLoad = () => {
+        // TODO(karliatto): this event listener is actually not usable since
+        // connect is dispose in onLoad init below in this file when
```

Comment:
what do you mean?  that it disappears after reload?  or something else?

Reply:
As you can see few lines below, there is ` TrezorConnect.dispose();` and that cancel all the listeners that were registered before, it seams.